golem_messages/message/tasks.py

```diff
@@ -57,18 +57,46 @@ def _get_task_value(self, attr_name):
                 if hasattr(self, slot)]
 
         for msg in msgs:
-            if hasattr(msg, attr_name):
-                return getattr(msg, attr_name)
+            val = getattr(msg, attr_name, None)
+            if val:
+                return val
         return None
 
     @property
     def task_id(self):
+        """
+        :return: the `task_id` related to this message chain
+        """
         return self._get_task_value('task_id')
 
     @property
     def subtask_id(self):
+        """
+        :return: the `subtask_id` related to this message chain
+        """
         return self._get_task_value('subtask_id')
 
+    @property
+    def ttc(self):
+        """
+        :return: the `TaskToCompute` related to this message chain
+        """
+        return self._get_task_value('ttc')
```

Comment:
Shouldn't this be called `task_to_compute`? Because that's the name of all ttc slots